packages/now-static-build/src/frameworks.ts

```diff
@@ -323,6 +323,18 @@ export const frameworks: Framework[] = [
     dependency: '@stencil/core',
     getOutputDirName: async () => 'www',
     defaultRoutes: [
+      { 
+        src: '/assets/(.*)',
+        headers: { 'cache-control': 'max-age=2592000' },
+      },
+      {
+        src: '/build/p-.*',
```

Comment:
Is this documented somewhere? What is `p`?

Reply:
I don't think it's documented (at least couldn't find it)... in production builds, all the bundles get an md5 hash as their name based on their content and are prefixed with `p-`. All files with that prefix can be cached as immutable, but there's also other files in the `/build` folder which don't start with `p-` and shouldn't be cached.  I don't actually know what `p` stands for, but maybe "production"? /cc @manucorporat

Reply:
I'll approve since @adamdbradley approved and works on stencil. 

However, I think you should document this behavior like [Gatsby](https://www.gatsbyjs.org/docs/caching/) so that this is a firm contract we can depend on.

Otherwise stencil deployments will start caching the wrong files and there is no way to undo once a client caches the wrong file.

Reply:
Also lets get @AndyBitz to review